iceoryx_posh/include/iceoryx_posh/internal/popo/trigger.inl

```diff
@@ -97,28 +97,6 @@ inline Trigger::Trigger(EventBasedTrigger_t,
               notificationTypeHash)
 {
 }
-
-template <typename T>
-inline void Trigger::updateOrigin(T& newOrigin) noexcept
```

Comment:
This wasn't needed anymore because `WaitSet` and `Listener` aren't moveable anymore. Hence, a `updateOrigin` method is not needed.